src/components/Widgets/Markdown/serializers/slateRemark.js

```diff
@@ -37,6 +38,108 @@ const markMap = {
   code: 'inlineCode',
 };
 
+let shortcodePlugins;
+
+export default function slateToRemark(raw, opts) {
+  /**
+   * Set shortcode plugins in outer scope.
+   */
+  ({ shortcodePlugins } = opts);
```

Comment:
Instead of mutating a global variable, would it be possible to give an additional optional `shortcodePlugins` argument to  `transform` and curry it when `transform` recurses,
 e.g. `flatMap(combinedChildren, node => transform(node, shortcodePlugins))`? I don't see anything in this file preventing that, but the "mimics the approach of a Remark plugin" comment makes me wonder if this is getting used somewhere else that requires that single-argument API.